app/src/main/java/ch/epfl/polychef/utils/Similarity.java

```diff
@@ -2,7 +2,23 @@
 
 import java.util.Locale;
 
+/**
+ * An utility class containing similarity methods.
+ */
```

Comment:
Maybe "A utility class computing the similarity between Strings"

Reply:
👍